tests/test_persistence.py

```diff
@@ -1633,6 +1345,50 @@ def test_save_on_flush_single_files(self, pickle_persistence, good_pickle_files)
             conversations_test = dict(pickle.load(f))['conversations']
         assert conversations_test['name1'] == conversation1
 
+    def test_custom_pickler_unpickler(
+        self, bot, update, pickle_persistence, good_pickle_files, recwarn
+    ):
+        u = UpdaterBuilder().bot(bot).persistence(pickle_persistence).build()
+        dp = u.dispatcher
+        bot_id = None
+
+        def first(update, context):
+            nonlocal bot_id
+            bot_id = update.message.get_bot()
+            assert bot_id
```

Comment:
I don't understand this one. `assert bot_id is bot`, right? what does this test?

Reply:
That was just testing if a bot is present in the message, but guess that's not needed, since get_bot() would fail when there is no bot anyway